wandb/apis/normalize.py

```diff
@@ -23,10 +39,16 @@ def wrapper(*args, **kwargs):
         message = "Whoa, you found a bug."
         try:
             return func(*args, **kwargs)
-        except requests.HTTPError as err:
-            raise CommError(err.response, err)
-        except ContextCancelledError as err:
```

Comment:
Where did this end up? context cancelled?

Reply:
There is now 
```
except Error as err:
            raise err
```
That is used for all errors originating from wandb.